visual-studio-dev-vm-O365/azuredeploy.json

```diff
@@ -0,0 +1,275 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.2",
+  "parameters": {
+    "storageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Standard_LRS"
+      ],
+      "metadata": {
+        "description": "Which type of storage you want to use"
+      }
+    },
+    "vmName": {
```

Comment:
Move to a variable, the updated BP is to move params to vars if the user doesn't need to specify for a particular reason - vm name can be defaulted.